barterdude/hooks/logging.py

```diff
@@ -1,24 +1,21 @@
-import logging
+from logging import getLogger
+from traceback import format_tb
```

Comment:
I like [tbutils](https://boltons.readthedocs.io/en/latest/tbutils.html) for this because it already returns a dict representation of error, but default traceback works too

Reply:
I will take a look.